Test/Unit/TestHelper.php

```diff
@@ -19,7 +19,9 @@ class TestHelper extends TestCase
      */
     public static function getReflectedClass($class)
     {
-        if (is_subclass_of($class, 'PHPUnit_Framework_MockObject_MockObject')) {
+        if (is_subclass_of($class, '\PHPUnit\Framework\MockObject\MockObject')) {
```

Comment:
In PHPUnit 9, the correct class name is \PHPUnit\Framework\MockObject\MockObject